core/roboconf-messaging-http/src/main/java/net/roboconf/messaging/http/internal/MessagingWebSocket.java

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright 2015 Linagora, Université Joseph Fourier, Floralis
+ *
+ * The present code is developed in the scope of the joint LINAGORA -
+ * Université Joseph Fourier - Floralis research program and is designated
+ * as a "Result" pursuant to the terms and conditions of the LINAGORA
+ * - Université Joseph Fourier - Floralis research program. Each copyright
+ * holder of Results enumerated here above fully & independently holds complete
+ * ownership of the complete Intellectual Property rights applicable to the whole
+ * of said Results, and may freely exploit it in any manner which does not infringe
+ * the moral rights of the other copyright holders.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.roboconf.messaging.http.internal;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Logger;
+
+import javax.websocket.ClientEndpoint;
+import javax.websocket.CloseReason;
+import javax.websocket.OnClose;
+import javax.websocket.OnError;
+import javax.websocket.OnMessage;
+import javax.websocket.OnOpen;
+import javax.websocket.Session;
+import javax.websocket.server.ServerEndpoint;
+
+import net.roboconf.messaging.api.utils.SerializationUtils;
+import net.roboconf.messaging.http.HttpMessage;
+import net.roboconf.messaging.http.SubscriptionMessage;
+
+/**
+ * @author Pierre-Yves Gibello - Linagora
+ */
+@ClientEndpoint
```

Comment:
Le 07/10/2015 11:25, Vincent Zurczak a écrit :
>
> In 
> core/roboconf-messaging-http/src/main/java/net/roboconf/messaging/http/internal/MessagingWebSocket.java 
> <https://github.com/roboconf/roboconf-platform/pull/459#discussion_r41368822>:
>
> > +import javax.websocket.CloseReason;
> > +import javax.websocket.OnClose;
> > +import javax.websocket.OnError;
> > +import javax.websocket.OnMessage;
> > +import javax.websocket.OnOpen;
> > +import javax.websocket.Session;
> > +import javax.websocket.server.ServerEndpoint;
> > +
> > +import net.roboconf.messaging.api.utils.SerializationUtils;
> > +import net.roboconf.messaging.http.HttpMessage;
> > +import net.roboconf.messaging.http.SubscriptionMessage;
> > +
> > +/**
> > + * @author Pierre-Yves Gibello - Linagora
> > + */
> > +@ClientEndpoint
>
> I'm not fond of these annotations...
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/roboconf/roboconf-platform/pull/459/files#r41368822>.
>
Neither am I... I use them for the tests.
Removing them (and skipping tests) does not solve anything.